Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n.idea/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 65c9f467b428f72fa386f7f5ada7095a9d4447db)
+++ b/.gitignore	(date 1707151097544)
@@ -1,2 +1,4 @@
 # Default ignored files
-.idea/
\ No newline at end of file
+.idea/
+.plot_functions.py
+.plot_functions_reformed.py
\ No newline at end of file
